treon/treon.py

```diff
@@ -127,4 +128,6 @@ def get_notebooks_to_test(args):
     if not result:
         sys.exit('No notebooks to test in {path}'.format(path=path))
 
+    for exclude_str in args['--exclude']:
+        result = [file for file in result if exclude_str not in file]
```

Comment:
If we accept full file & directory paths then we can have a stricter condition i.e. exclude from test if filename starts with exclude_str. 

Stricter condition is preferred to make sure only the intended files & directories are excluded.   Current pattern matching ("in" condition) can cause some unintended files to be skipped. E.g. I exclude "predictions.ipynb" from my test suite without knowing there is "final_important_predictions.ipynb" somewhere deeply nested.